docs/specs/output.yml

```diff
@@ -412,3 +412,65 @@ outputs:
       }
     }
 ---
+# output .links.json
+inputs:
+  docfx.yml: |
+    monikerRange:
+      'docs/v1/**': '< netcore-2.0'
+      'docs/v2/**': '>= netcore-2.0'
+    routes:
+      docs/v1/: docs/
+      docs/v2/: docs/
+    monikerDefinition: monikerDefinition.json
+  docs/v1/a.md: |
+    ---
+    uid: a
+    title: netcore
+    monikerRange: '> netcore-1.0'
+    ---
+    Moniker: netcore-1.1
+  docs/v1/b.md: |
+    Link to @a
+    Link to [some text](../v2/c.md)
+    Link to [not existed](c.md)
+    Link to [absolute link](/docs/c)
```

Comment:
>to [](start = 9, length = 2)

what about link to empty like [empty]()

Reply:
We would not output this since a non-null/empty link check before adding them